inspirehep/utils/stats.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2016 CERN.
+#
+# INSPIRE is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+import operator
+
+
+class Stats(object):
```

Comment:
I agree. I was being 'pythonic' whose only known synonym is 'convoluted'.

On Wed, 15 Jun 2016, 17:31 Javier Martin Montull, <notifications@github.com>
wrote:

> In inspirehep/utils/stats.py
> <https://github.com/inspirehep/inspire-next/pull/1218#discussion_r67185130>
> :
>
> > +# modify it under the terms of the GNU General Public License as
> > +# published by the Free Software Foundation; either version 2 of the
> > +# License, or (at your option) any later version.
> > +#
> > +# INSPIRE is distributed in the hope that it will be useful, but
> > +# WITHOUT ANY WARRANTY; without even the implied warranty of
> > +# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
> > +# General Public License for more details.
> > +#
> > +# You should have received a copy of the GNU General Public License
> > +# along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
> > +# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
> > +import operator
> > +
> > +
> > +class Stats(object):
>
> If a class is not needed we can consider just having functions that
> receive the needed parameters instead of *args, **kwargs
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/inspirehep/inspire-next/pull/1218/files/c1a56e65e5ce7b6955d4913e12a02d23e9355d83#r67185130>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AARPHDT9Tj8CiOrKmfrh_kgJjLlHpOo6ks5qMBrngaJpZM4I2d4f>
> .
>